src/west/app/project.py

```diff
@@ -161,6 +161,8 @@ def do_add_parser(self, parser_adder):
         parser.add_argument('--mr', '--manifest-rev', dest='manifest_rev',
                             help='''manifest revision to check out and use;
                             cannot be combined with -l''')
+        parser.add_argument('--mf', '--manifest-file', dest='manifest_file',
+                            help='''manifest file name to use''')
```

Comment:
A minor point, but if you have a chance, there's no need to triple-quote a single line string, so this could just use single quotes.